addons/hr/views/hr_work_location_views.xml

```diff
@@ -8,9 +8,8 @@
                 <list string="Work Location">
                     <field name="active" column_invisible="True" />
                     <field name="name" />
-                    <field class="o_homework_icon_types d-flex flex-wrap" name="location_type"
-                           widget="hr_homeworking_radio_image" options="{'horizontal': true}"/>
```

Comment:
Was it in the spec to remove this from the list view? If yes, then I think we can remove that component (hr_homeworking_radio_image) files, as both uses are removed (+add a line in the commit message)